torchao/dtypes/fbgemm_fp8_tensor.py

```diff
@@ -146,6 +155,43 @@ def _(func, types, args, kwargs):
     )
 
 
+@implements(aten.slice.Tensor)
+def _(func, types, args, kwargs):
+    self, dim, start, end, step = fill_defaults(args, 5, [0, None, None, 1])
```

Comment:
I think this stuff is error prone, do you think that : https://github.com/pytorch/ao/blob/488ecd4cf871035b091981a557a9204a5d50bf5b/torchao/dtypes/floatx/float8_layout.py#L302

is helpful here?

Reply:
the scale here is not 2d actually, but might be useful if we resize that to 2D/3D

Reply:
I dont think this relies on 2d vs 3d no?

Reply:
does this assume scale has the same shape as data: https://github.com/pytorch/ao/blob/488ecd4cf871035b091981a557a9204a5d50bf5b/torchao/dtypes/floatx/float8_layout.py#L321